ietf/group/utils.py

```diff
@@ -120,6 +120,28 @@ def can_manage_group(user, group):
         return True
     return group.has_role(user, group.features.groupman_roles)
 
+def groups_managed_by(user, group_queryset=None):
+    """Find groups user can manage"""
+    if group_queryset is None:
+        group_queryset = Group.objects.all()
```

Comment:
No - Python evaluates the expressions in the definition once at import, which may happen before Django is officially ready. More generally, default values that are mutable are a major no-no in Python because they'll leak data between calls to the method.